tests/test_RemoveLinksToAttachments.py

```diff
@@ -0,0 +1,83 @@
+"""
+Tests the RemoveLinksToAttachments script for 4 different cases. First creates a test database with 4 tables, populated
+with test data to be passed into the original script. Tests are preformed to confirm that attachmentID's with mimeType
+'image/jpeg' are selected properly, that attachmentID's that create no conflicts are removed properly, that when the
+function is called with no conflicts an empty conflict list is returned, and that when the function is called with
+conflicts a non empty conflict list is returned
+"""
+import RemoveLinksToAttachments, os, test_db, sqlite3, unittest
```

Comment:
what is the test_db module? does this get used anywhere?

Reply:
the test_db does not get used and has been removed